terminus-ui/selection-list/src/selection-list-panel/selection-list-trigger.directive.ts

```diff
@@ -752,7 +770,8 @@ export class TsSelectionListTriggerDirective<ValueType = string> implements Cont
         // Create a new stream of panelClosingActions, replacing any previous streams that were created, and flatten it so our stream only
         // emits closing events...
         switchMap(() => {
-          this.resetActiveItem();
+          // Focus the first option when options change
+          this.selectionListPanel.keyManager.setActiveItem(0);
```

Comment:
Does `resetActiveItem` also do the same thing - set the first option active? although, this way is clearer...

Reply:
`resetActiveItem` clears the selection completely. Which mentally seems fine until you start using the component. The UX of being able to just type to filter and hit `enter` is just better imo